packages/@jsii/spec/package.json

```diff
@@ -31,7 +31,8 @@
     "package": "package-js"
   },
   "dependencies": {
-    "ajv": "^8.11.0"
+    "ajv": "^8.11.0",
+    "fs-extra": "^10.1.0"
```

Comment:
hoping this addition is ok. fs-extra is used for `writeJsonSync` and `readJsonSync` and i'm sure I could just reimplement those if we don't want to add dependencies.

Reply:
It's okay - `fs-extra` is widely used in our codebases already. It's true these can be trivially achieved with `JSON.stringify`/`JSON.parse` and `fs.writeFileSync`/`fs.readFileSync`... but let's not overthink it.